src/AcceptanceTests.SQS/SetupFixture.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using System;
+    using System.Text.RegularExpressions;
+    using System.Threading.Tasks;
+    using Amazon.S3;
+    using Amazon.SimpleNotificationService;
+    using Amazon.SQS;
+    using NUnit.Framework;
+    using Transport.SQS.Tests;
+
+    [SetUpFixture]
+    public class SetupFixture
```

Comment:
@TravisNickels  Not sure if this will be invoked? we might have to find a way for the shared tests to allow this stuff to be invoked somehow